includes/snmp.inc.php

```diff
@@ -264,6 +282,29 @@ function snmp_get($device, $oid, $options = null, $mib = null, $mibdir = null)
 }//end snmp_get()
 
 
+function snmp_get_next($device, $oid, $options = null, $mib = null, $mibdir = null)
```

Comment:
I am only using the snmp_get_next to traverse the IF-MIB::ifIndex
Once I have my indexes I use the index to poll the various OIDs, eg genEquipRadioStatusDefectedBlocks.268451969, genEquipRadioStatusDefectedBlocks.268451970.

The other option I can see is like this:

if (IP10) {
    get genEquipRadioStatusDefectedBlocks.1
} else if (IP20C) {
    get genEquipRadioStatusDefectedBlocks.268451969
    get genEquipRadioStatusDefectedBlocks.268451970
} elseif (IP20S) {
    get genEquipRadioStatusDefectedBlocks.?????
} elseif (IP20N) {
    get genEquipRadioStatusDefectedBlocks.?????
    get genEquipRadioStatusDefectedBlocks.?????
    get genEquipRadioStatusDefectedBlocks.?????
    get genEquipRadioStatusDefectedBlocks.?????
} elseif ...


I'm trying to make the discovery and polling as generic (and extensible) as possible for various variants of ceragon radios since I don't have all of them to test against.